bugbug/models/uplift.py

```diff
@@ -56,6 +56,9 @@ def __init__(self, lemmatization=False):
         self.clf = xgboost.XGBClassifier(n_jobs=16)
         self.clf.set_params(predictor='cpu_predictor')
 
+    def rollback(self, change):
+        return (change['field_name'].startswith('flagtypes.name') and change['added'].startswith('approval-mozilla-') and (change['added'].endswith('+') or change['added'].endswith('-')))
```

Comment:
Sorry, one last thing.
For `change['field_name'].startswith('flagtypes.name')`, I think it is unnecessary to use `startswith`, it can be just an equality check. Could you confirm? (to confirm, just check that in all histories of all bugs there is no case where change['field_name'].startswith('flagtypes.name') but change['field_name'] != 'flagtypes.name'.

Reply:
checked and there is not such case 👍